### ISSUE TITLE
[MU4 Issue] Add new instruments incl. bagpipes and Chinese orchestral

### ISSUE THREAD

**Issue by Reloup38** [musescore/MuseScore#13987]
**Describe the bug**
Instruments that aren't native to the instrument list in musescore aren't displayed in the mixer when added to a score, no matter if it is in the instrument list 1 or 2 space.

**To Reproduce**
1. make a custom instrument that doesn't use the name of any of the "vanilla" instruments
2. load the instrument list into musescore
3. add the instruments to musescore (here, i added the flute to show "vanilla" instruments are still showing correctly)
4. open the mixer

**Expected behavior**
Every instrument in your score should be in the mixer.  The fact that they aren't make it so they don't produce any playback sound.

**Screenshots**
![image](https://user-images.githubusercontent.com/116571960/197578270-31701235-35ee-48d8-a437-2fa9bc86c429.png)

**Platform information**
 - OS: Windows 7

**Additional context**
I modified the instrument list to add a lot of lacking or obscure instruments that i use a lot in my writing. I'd be glad if the instruments i made code for could be added to musescore, i'd be willing to share them for free.

**Comment by Tantacrul** [comment]
Duplicate of https://github.com/musescore/MuseScore/issues/13910

**Comment by Tantacrul** [comment]
It would be good to know what instruments you are missing? That way we can add them to our proper list.

**Comment by shoogle** [comment]
@Reloup38, yes please let us know what instruments you need. You can attach your custom instruments.xml to a comment here if that's easier.

**Comment by Reloup38** [comment]
@shoogle @Tantacrul 
Here are the instruments i added in my instrument.xml. these includes :
- A bunch of missing instrument sizes, discovered thanks to bandestration.com by Bret Newton and the good folks in the Subcontrabassoon discord server, i've also included some of the instruments they have worked on.
- A few chinese orchestral instruments thanks to Hong Kong Chinese Orchestra
- Missing sizes of steel pans
- Bagpipes
- New violin family
- and a bit more

I can understand if you don't add all of them, especially the percussion ones as they require very special soundfonts. I'll let you decide what to add or not. A few of these instruments can be directly copy-pasted into an instrument.xml file, but a lot of them probably need to be cleaned up first

I'd be glad to be part of the great adventure that is musescore.
[instruments 3_2.zip](https://github.com/musescore/MuseScore/files/9855076/instruments.3_2.zip)

**Comment by MarcSabatella** [comment]
Would be good to reopen this now that we are past the 4.0 and into the 4.X
